en_us.lang

```diff
@@ -157,200 +157,200 @@ gui.xaero_transfer_all=Transfer All Waypoints
-gui.xaero_toggle_map_waypoints=Toggle On-map Waypoints
-gui.xaero_entity_nametags=Entity Name Tags
+gui.xaero_box_distance=Specifies, for which in-game waypoints to show the distance to.
+gui.xaero_box_always_distance=Display distance even when the waypoint is very close. This setting does not override the "Distance to WP" option.
+gui.xaero_box_arrow_scale=Scale of the arrow used in the non-rotating variant of the minimap and several other cases.
+gui.xaero_box_arrow_color=Color of the arrow used in the non-rotating variant of the minimap and several other cases.
+gui.xaero_box_lock_player_heads=Display player heads instead of colored dots even if the Tab key is not pressed.
+gui.xaero_box_cave_zoom=How much to zoom further in the minimap when the cave mode is active.
+gui.xaero_box_entity_depth=Display entities darker on the minimap depending on their Y level relative to you.
+gui.xaero_box_height_limit=Maximum Y-level distance for entities until they are no longer displayed on the minimap.
+gui.xaero_box_compass_over_wp=Render compass letters (N, E, S, W) over the on-map waypoints.
+gui.xaero_toggle_map_waypoints=Toggle On-Map Waypoints
+gui.xaero_entity_nametags=Entities With Name Tags
 gui.xaero_minimap_text_align=Info Text Alignment
 gui.xaero_center=Center
 gui.xaero_left=Left
 gui.xaero_right=Right
-gui.xaero_show_angles=Show View Angles
-gui.xaero_box_entity_nametags=Display entity name under the dot for entities that have a name tag.
-gui.xaero_box_teleport_default_command=The command used for waypoints teleportation if a server-specific command is not set in the waypoints menu Options screen.
-gui.xaero_waypoints_edit_mode_individually=One-by-one
-gui.xaero_waypoints_edit_mode_all=All
+gui.xaero_show_angles=Display Facing Angles
+gui.xaero_box_entity_nametags=Display entity name under the dot on the minimap for entities that have name tags.
+gui.xaero_box_teleport_default_command=The command used for waypoint teleportation if a world-specific or a server-specific command is not set in the Options menu on the waypoint screen.
+gui.xaero_waypoints_edit_mode_individually=One by One
+gui.xaero_waypoints_edit_mode_all=All at Once
 gui.xaero_compass=Compass
-gui.xaero_cave_maps_depth=Cave Maps Depth
+gui.xaero_cave_maps_depth=Cave Mode Depth
 gui.xaero_waypoints_edit_reset=Reset
 gui.xaero_main_entity_as=Main Entity As
 gui.xaero_arrow=Arrow
 gui.xaero_dot=Dot
 gui.xaero_crosshair=Crosshair
-gui.xaero_teleport_coordinates_hidden=Unable to teleport because it would tell you the waypoint coordinates. Disable "Hide Waypoint Coords" to be able to freely teleport again.
-gui.xaero_hide_wp_coords=Hide Waypoint Coords
+gui.xaero_teleport_coordinates_hidden=Unable to teleport, because it could expose the waypoint coordinates. Disable the "Hide Waypoint Coordinates" option to be able to freely teleport again.
+gui.xaero_hide_wp_coords=Hide Waypoint Coordinates
```

Comment:
What do you think about adding a tooltip to the button that has this label? I suppose that there may be a group of people (particularly streamers) who don't know what this option does, but would benefit if they knew it.

Reply:
IDK. It's so obvious what it does, isn't it?
This doesn't fit the button, btw.

Reply:
It depends, I think. Some time ago I had a longer break from modded Minecraft, and when I came back to your minimap mod and saw the "Hidden Waypoint Coords" option, I started to wonder what it does, and after a longer while I realized that I should have already know about this option, because I had read about it in the changelog. Most people don't read changelogs at all, so the button label alone may be insufficient. What do you think?

Reply:
But the option literally hides waypoint coordinates (hopefully everywhere). Were you confused about where it would hide them?

Reply:
I'd like to help, but it happened many months ago, and now I don't remember what was the problem at that time. If you feel this button label is self-explanatory and doesn't need an additional clarification, feel free to leave it as is.

Reply:
Yeah, I think it's fine!